Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -0,0 +1,91 @@
+/// Describes the modes for configuring the fullDocument field of a `ChangeStreamDocument`.
+public enum FullDocument: RawRepresentable, Codable {
+    /// Specifies that the `fullDocument` field will contain a copy of the entire document that was changed from
+    /// some time after the change occurred. If the document was deleted since the updated happened, it will be null.
+    case updateLookup
+    /// For an unknown value. For forwards compatibility, no error will be thrown when an unknown value is provided.
+    case other(String)
+
+    public var rawValue: String {
+        switch self {
+        case .updateLookup:
+            return "updateLookup"
+        case .other(let v):
+            return v
+        }
+    }
+
+    public init?(rawValue: String) {
+        switch rawValue {
+        case "updateLookup":
+            self = .updateLookup
+        default:
+            self = .other(rawValue)
+        }
+    }
+}
+
+/// Options to use when creating a `ChangeStream`.
+public struct ChangeStreamOptions: Codable {
+    /**
+     * Indicates how the `fullDocument` field of a `ChangeStreamDocument` should be filled out by the server.
```

Comment:
nit: I'd suggest just saying "change stream document" rather than `ChangeStreamDocument` as it will also apply to the documents returned in the case where you have a `ChangeStream<Document>`